web/pandas/_templates/layout.html

```diff
@@ -91,7 +91,7 @@
                 </li>
             </ul>
             <p>
-                pandas is a fiscally sponsored project of <a href="https://numfocus.org">NumFOCUS.</a>
+                © 2022 pandas via <a href="https://numfocus.org">NumFOCUS.</a>
```

Comment:
Can you use `NumFOCUS, Inc.` instead please, that's the legal name.

For the year, can you make it be generated dynamically, so we don't need to update it every year? I'm not sure if jinja allows direct access to the `datetime` module, or what the best option could be. Feel free to add the year to the context if you think that's the best, or also use javascript

Reply:
Thanks for the feedback, I'll try soon!